cmd/seed-controller-manager/options.go

```diff
@@ -145,6 +149,9 @@ func newControllerRunOptions() (controllerRunOptions, error) {
 	flag.StringVar(&c.lokiRulerURL, "loki-ruler-url", "http://loki-distributed-ruler.mla.svc.cluster.local:3100", "The URL of loki ruler which is running for MLA stack.")
 	flag.StringVar(&c.machineControllerImageTag, "machine-controller-image-tag", "", "The Machine Controller image tag.")
 	flag.StringVar(&c.machineControllerImageRepository, "machine-controller-image-repository", "", "The Machine Controller image repository.")
+	flag.StringVar(&c.operatingSystemManagerImageTag, "operating-system-manager-image-tag", "", "The Operating System Manager image tag.")
+	flag.StringVar(&c.operatingSystemManagerImageRepository, "operating-system-manager-image-repository", "", "The Operating System Manager image repository.")
+
```

Comment:
Why do we need these as new CLI flags? All controllers should have access to the KubermaticConfiguration and can react dynamically to it changing. No need to redeploy the seed-ctrl-mgr when something is changed. Or am I overlooking something?

In general, we want to _reduce_ the number of CLI flags over time, not grow them. ^^

Reply:
Done